### ISSUE TITLE
new save button

### ISSUE THREAD

**Issue by kfarr** [3DStreet/3dstreet-editor#60]
Instead of the "PNG Screenshot" option in Save menu it should say "3DStreet JSON"
and then trigger the `convertToObjection` function from https://github.com/3DStreet/3dstreet/blob/main/src/json-utils.js and save the resultant string as a file named streetname.3dstreet.json

only do this after #59 is completed
figma: https://www.figma.com/file/W9jid3A0jgssnIKBFEvG38/3DStreet-2022?node-id=2082%3A21414

![Image](https://user-images.githubusercontent.com/470477/199169530-50afa5a4-4afe-4805-a0a2-2ce00c1ae3e5.png)

**Comment by Alina-Kuntsiv** [comment]
We have two questions regarding this task:
1. Which entities to pass to convertToObject? (currently passing all via querySelectorAll('a-entity'));
2. I can't convert to html the elements that are formed from entities in this function. This is impossible due to looping of links on each other. Solution options do not fit here, because it does not coincide with the task. It is necessary to clarify what and in what format this function should return, and what and in what format to download.

**Comment by kfarr** [comment]
For (1) the entity to be saved should be `<a-entity id="streets" data-layer-name="Street Segments" data-layer-show-children>` and all of its children, getElementById('streets') for example

For (2) not sure I understand "convert to html". The output of convertToObject is an object. Then `JSON.stringify(output)` to create a string and save the string as a file with `.json` extension prompted for download by browser.

**Comment by omedvediev** [comment]
Hello, @kfarr! Thank you! The answer for the first question solved the entire problem.

**Comment by kfarr** [comment]
I had hoped to use the original json-utils.js function from 3dstreet instead of copying this over. I am doing some small changes to import the json-utils.js in this branch, so let's pause on this until I finish merging that and then we can revisit.
https://github.com/3DStreet/3dstreet-editor/tree/match-3dstreet-index-html

**Comment by kfarr** [comment]
in progress here: https://github.com/3DStreet/3dstreet-editor/pull/79
new issues tracked by new ticket here: https://github.com/3DStreet/3dstreet/issues/226
